eth/fetcher/fetcher_test.go

```diff
@@ -178,19 +178,21 @@ func (f *fetcherTester) makeBodyFetcher(peer string, blocks map[common.Hash]*typ
 	// Create a function that returns blocks from the closure
 	return func(hashes []common.Hash) error {
 		// Gather the block bodies to return
+		headerHashes := make([]common.Hash, 0, len(hashes))
```

Comment:
nit: `blockHashes` if you take the suggestion above

edit: And everywhere else, ofc